Riot/Modules/Room/Views/Title/Expanded/ExpandedRoomTitleView.m

```diff
@@ -66,16 +66,15 @@ - (void)refreshDisplay
         
         // Compute active members count
         MXWeakify(self);
-        [self.mxRoom members:^(MXRoomMembers *roomMembers) {
+        void (^onRoomMembers)(MXRoomMembers *roomMembers, BOOL allMembers) = ^void(MXRoomMembers *roomMembers, BOOL allMembers)
```

Comment:
Note: `allMembers` is not used

Reply:
I know. We decided to do nothing for the moment to show that we are fetching data from the HS. This code is just ready in case of UI update.